db/mapper.go

```diff
@@ -46,15 +48,19 @@ func (um utxoMapper) AddUTXO(ctx sdk.Context, utxo types.UTXO) {
 	position := utxo.GetPosition()
 	pos := um.encodePosition(position)
 
+	addr := utxo.GetAddress().Bytes()
+	pos = append(addr, pos...)
+
 	store := ctx.KVStore(um.contextKey)
 	bz := um.encodeUTXO(utxo)
 	store.Set(pos, bz)
 }
 
 // Deletes UTXO corresponding to the position from mapping
-func (um utxoMapper) DeleteUTXO(ctx sdk.Context, position types.Position) {
+func (um utxoMapper) DeleteUTXO(ctx sdk.Context, addr common.Address, position types.Position) {
 	store := ctx.KVStore(um.contextKey)
 	pos := um.encodePosition(position)
+	pos = append(addr.Bytes(), pos...)
 	store.Delete(pos)
 }
 
```

Comment:
Add helper function to take address and position and output store key.

Add function in mapper to take address and return all UTXO's that belong to it. This can be done with KVStorePrefixIterator.

Reply:
Add a comment above the helper function showing <address> + <encoded position> is key for mapping to utxo

Reply:
The function described above will probably need to be accessible via the ChildChain struct right?